File: Arcade/Intro/Java/ArrayMaximalAdjacentDifference.java
import java.util.stream.IntStream;

final class ArrayMaximalAdjacentDifference {
	
	int arrayMaximalAdjacentDifference(int[] inputArray) {    
	    return IntStream.range(0, inputArray.length - 1)
	    .map(i -> Math.abs(inputArray[i] - inputArray[i + 1])).max().getAsInt();
	}
}

File: Arcade/Intro/javascript/arrayMaximalAdjacentDifference.js
function arrayMaximalAdjacentDifference(inputArray) {
    let max = 0;
    for(let i = 0; i < inputArray.length - 1; i++) {
        let t = Math.abs(inputArray[i] - inputArray[i + 1]);
        if(t > max) max = t;
    }
    return max;
}